### ISSUE TITLE
Update DNSSEC related pages

### ISSUE THREAD

**Issue by ggalmazor** [dnsimple/dnsimple-support#657]
Belongs to https://github.com/dnsimple/dnsimple-business/issues/1195

We need to review information and language related to DNSSEC:
- [ ] https://support.dnsimple.com/articles/before-transferring-domain/
- [ ] https://support.dnsimple.com/articles/transferring-domain/
- [ ] https://support.dnsimple.com/articles/ds-records-changing-dns/
- [ ] https://support.dnsimple.com/articles/cloudflare-ds-record/
- [ ] https://support.dnsimple.com/articles/dnssec/

**Comment by weppos** [comment]
Se also https://github.com/dnsimple/dnsimple-support/issues/725 for a more specific individual need.

**Comment by ggalmazor** [comment]
We should consider the research we did at https://github.com/dnsimple/dnsimple-app/issues/14595, which revealed a scenario where deleting the DS record would not be appropriate.

**Comment by weppos** [comment]
> We should consider the research we did at [dnsimple/dnsimple-app#14595](https://github.com/dnsimple/dnsimple-app/issues/14595), which revealed a scenario where deleting the DS record would not be appropriate.

We can reconsider our choices if necessary. For now, I would at minimum make sure our documentation reflects (or documents) what we are offering.

**Comment by ggalmazor** [comment]
Agreed. I only wanted to bring your attention to a scenario where we should not instruct our customers to delete their DS records:

> 2. The domain is delegated through a mix of DNSimple and 3rd party name servers
>
> We need to leave untouched any provisioned DS record that's not linked to an existing DNSSEC key set in our systems because it could be linked to the private key being used at the 3rd party name servers to produce RRSet signatures.

**Comment by deandre** [comment]
@weppos @ggalmazor can you please clarify what information needs review/update on those pages?

**Comment by ggalmazor** [comment]
I'd have to review these pages. I believe we changed them recently and maybe this issue is out of sync. Is there any deadline we want to hit, @deandre ?

**Comment by deandre** [comment]
> I'd have to review these pages. I believe we changed them recently and maybe this issue is out of sync. Is there any deadline we want to hit, @deandre ?

No particular deadline, just looking for clarification on what's needed from me. Thanks @ggalmazor!

**Comment by stephanoodle** [comment]
In [After the Transfer](https://support.dnsimple.com/articles/transferring-domain/#after-the-transfer), is there a reason why a customer would need to disable DNSSEC? At this point, shouldn't we be prompting the customer to make sure the name servers are changed and then enable DNSSEC? 

This seems confusing.

**Comment by weppos** [comment]
> In [After the Transfer](https://support.dnsimple.com/articles/transferring-domain/#after-the-transfer), is there a reason why a customer would need to disable DNSSEC? At this point, shouldn't we be prompting the customer to make sure the name servers are changed and then enable DNSSEC?
> 
> This seems confusing.

Yes, you are correct. _You should disable DNSSEC before changing the name server delegation_, as while transitioning from one DNS provider to another (aka to us), because it's currently not possible to retain the same signing setting during the delegation change.

It is also recommended you turn it off before transferring the domain to us. While we will inherit any DS record attached to the domain once we take it over, occasionally some stale DS records remain around.

The TL;DR is that you should disable DNSSEC before migrating the domain registration, changing domain delegation, or both.

I think the "After the transfer" advice currently refers to the fact we have a section called _Changing name servers_ that reminds to switch the name server, if necessary. But the heading formatting is wrong, so if we want to keep the "disable DNSSEC" advice it should really be a warning within the "changing name servers" section.

**Comment by ggalmazor** [comment]
@SarahCaminiti1, after taking a trip to the past, here's what I can tell you about this ticket:
- This ticket belongs to https://github.com/dnsimple/dnsimple-business/issues/1195, which I closed on Oct 26th, 2021.
- The business ticket had a completed task about reviewing all articles and ensuring all were up to date.
- This ticket was created on Sep 29th *2020* (prior to the project's completion).

From what I can gather - especially the fact that the business ticket has completed tasks about support articles and blog posts, this ticket is just some leftovers I failed to keep updated or close once the business project was done. 

Since then (Oct 26th, 2021), I haven't been so involved in DNSSEC, and other teams have taken the lead, but as far as I'm aware, all support articles are up to date (beyond any possible issues discovered by Steph [here](https://github.com/dnsimple/dnsimple-support/issues/657#issuecomment-1549368800)).

Let's close this ticket as stale and open new tickets for specific amends we want to introduce if needed.

WDYT?
